engine/resource/src/resource_chunk_cache.cpp

```diff
@@ -54,9 +55,10 @@ struct ResourceChunkCache
     // The array is sorted on (path_hash, offset)
     dmArray<ResourceInternalDataChunk*> m_Chunks;
 
-    uint32_t m_CacheSize;    // The cache size for all streaming sounds
-    uint32_t m_CacheSizeUsed;// The amount of cache currently used
-    DLList   m_LRU; // head is MRU, tail is LRU
+    uint32_t m_CacheSize;       // The cache size for all streaming sounds
+    uint32_t m_CacheSizeUsed;   // The amount of cache currently used
+    DLList   m_LRU;             // head is MRU, tail is LRU
+    DLList   m_LRUNoEvict;      // head is MRU, tail is LRU
```

Comment:
I decided it was easier to maintain, and also avoid performance hits when keeping them all in the same LRU.
Each chunk (which is-a list node), can only be in one list at a time.